spinn_front_end_common/utility_models/reverse_ip_tag_multicast_source_machine_vertex.py

```diff
@@ -224,7 +224,8 @@ def __init__(
             self._install_virtual_key(n_keys)
 
     def _install_send_buffer(self, n_keys, send_buffer_times, target_address):
-        if send_buffer_times and hasattr(send_buffer_times[0], "__len__"):
+        if (send_buffer_times is not None and
```

Comment:
This must have slipped in from a previous linting.  It should be if len(send_buffer_times) and hasattr... as this is checking whether send_buffer_times has some values in it.  A prior check already checks for None.